internal/application/workload.go

```diff
@@ -205,14 +205,13 @@ func (a *Workload) Get(ctx context.Context) (*models.AppDeployment, error) {
 		routes = []string{err.Error()}
 	}
 
-	var status string
+	// -- errors retrieving the pod metrics are ignored.
+	// -- this will be reported later as `not available`.
+	// note: The pod metrics are not nil in that cases, just an empty slice.
+	// that is good, as that allows AFP below to still generate the basic pod info.
+	podMetrics, _ := a.getPodMetrics(ctx)
```

Comment:
Same, I would at least log it. It's not a critical error but it could be something else.

Reply:
Done, commit 08057f81b20e3de68c33f79b6f4055a66e2fcbb9